pkg/driver/nri_hooks.go

```diff
@@ -91,16 +96,19 @@ func (np *NetworkDriver) RunPodSandbox(ctx context.Context, pod *api.PodSandbox)
 	start := time.Now()
 	defer func() {
 		klog.V(2).Infof("RunPodSandbox Pod %s/%s UID %s took %v", pod.Namespace, pod.Name, pod.Uid, time.Since(start))
+		nriPluginRequestsLatencySeconds.WithLabelValues(methodRunPodSandbox).Observe(time.Since(start).Seconds())
 	}()
 	// get the devices associated to this Pod
 	podConfig, ok := np.podConfigStore.GetPodConfigs(types.UID(pod.GetUid()))
 	if !ok {
+		nriPluginRequestsTotal.WithLabelValues(methodRunPodSandbox, statusFailed).Inc()
```

Comment:
ditto, it does not find the config , IIUIC does not mean is failed

Reply:
Updated